projects/eudsl-python-extras/mlir/extras/util.py

```diff
@@ -460,3 +460,14 @@ def __call__(self, *args, **kwargs):
         self.kwargs = kwargs
         self.args = args
         return self
+
+
+def make_nanobind_metaclass_inheritable():
+    # Hack to allow us to inherit from nanobind's metaclass type
+    # https://github.com/wjakob/nanobind/pull/836
+    from .. import ir
```

Comment:
Nit: I don't think we need these to be local imports? If not then I'd prefer they were top level so that if we move this file we'll catch it even with exercising the function.